components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -31,13 +119,19 @@ BravePrefProvider::BravePrefProvider(PrefService* prefs,
               info->type(), prefs_, &brave_pref_change_registrar_,
               info->pref_name(),
               is_incognito_,
-              base::Bind(&PrefProvider::Notify, base::Unretained(this)))));
-      return;
+              base::Bind(&PrefProvider::Notify, weak_factory_.GetWeakPtr()))));
```

Comment:
it is safe to keep `Unretained` here (since we own `content_settings_prefs_`), so weak factory is not needed

Reply:
yea, I think this is also leftover from a previous iteration. I'll remove